Makefile

```diff
@@ -23,6 +26,7 @@ path_up_gateway: ## Run just the PATH gateway without any dependencies
 path_up_build_gateway: ## Run and build just the PATH gateway without any dependencies
 	docker compose up -d --build --no-deps path_gateway
 
+# TODO_UPNEXT(@adshmh): update path_up and path_down to use Tilt, and remove docker compose
```

Comment:
I ACK this TODO exists, but when I run `tilt up`, I couldn't get it to work, so how can I verify that it works?

Note: I could debug/fix it but didn't spend any time on it.

Strongly in favor that this has to be done IN THIS PR.

<img width="1840" alt="Screenshot 2024-11-11 at 5 36 07 PM" src="https://github.com/user-attachments/assets/e8db1b3d-d22d-418f-bb92-9af6afc17e10">

Reply:
I have updated this in a subsequent PR. Perhaps we could just add to this one?

https://github.com/buildwithgrove/path/blob/envoy-auth-tiltfile/Makefile#L21

```go
# TODO_NEXT(@commoddity): Implement MODE to enable path only or path + envoy auth containers
.PHONY: path_up_gateway
path_up_gateway: ## Run just the PATH gateway without any dependencies
	MODE=path_gateway tilt up

.PHONY: path_up
path_up: ## Run the PATH gateway and all related dependencies
	tilt up

.PHONY: path_down
path_down: ## Stop the PATH gateway and all related dependencies
	tilt down
```

Reply:
Thank you for the reviews, fixed.

> I have updated this in a subsequent PR. Perhaps we could just add to this one?

Thanks, I had to make changes locally so I ended up updating the Makefile.